assets/src/edit-story/app/story/effects/useLoadStory.js

```diff
@@ -41,37 +41,46 @@ function useLoadStory({ storyId, shouldLoad, restore }) {
         const {
           title: { raw: title },
           status,
-          author,
           slug,
           date_gmt,
           modified,
           excerpt: { raw: excerpt },
           link,
           story_data: storyDataRaw,
-          featured_media_url: featuredMediaUrl,
```

Comment:
If you are going to remove this @swissspidy, it should be removed from REST API all together.

https://github.com/google/web-stories-wp/blob/df38d616a7d6f410da5a86cd5c1e96d65411a444/includes/REST_API/Stories_Base_Controller.php#L127-L130

Reply:
@spacedmonkey yup 👍 I'll open a ticket for it